q2_types/_feature_data.py

```diff
@@ -153,18 +161,20 @@ def tabular_msa_to_aligned_dna_sequences(view, data_dir):
 plugin.register_data_layout(taxonomy_data_layout)
 plugin.register_data_layout_reader('taxonomy', 1, pd.Series,
                                    taxonomy_to_pandas_series)
+plugin.register_data_layout_reader('taxonomy', 1, pd.DataFrame,
+                                   taxonomy_to_pandas_dataframe)
 plugin.register_data_layout_reader('taxonomy', 1, qiime.Metadata,
                                    taxonomy_to_qiime_metadata)
```

Comment:
It isn't very useful right now because we haven't done anything fancy yet. The original idea was to just use `pd.DataFrame`, but we've been bitten by that before so we (@gregcaporaso, @ebolyen) wrote a light wrapper. It does serve as a kind of "token" to indicate that viewing this as metadata makes sense. (You could imagine `pd.DataFrame` would be a different result than the wrapped metadata view. In other words `qiime.core.type.Metadata` maps only to `qiime.Metadata`.